### DISCUSSION TITLE
Prepend/Append action under a condition

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#1164] [General]
Is it possible to prepend/append an action under a certain condition? Let's say a post is already published then I want to just display an unpublish action instead. Is it possible? Thanks

**Comment by danharrin** [comment]
Yeah, you can pass a callback to `hidden()` that receives `$record`. You can decide to hide or show the action there.

**Reply by ghost** [reply]
That's neat! I thought it was `visible()`. Thanks!

**Reply by danharrin** [reply]
I'd accept a PR for that too :)
